packages/components/organisms/f-loyalty/src/components/Header.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+    <div :class="$style['c-loyaltyHeader']">
+        <div :class="$style['c-loyaltyHeader-container']">
+            <bread-crumbs
+                :links="links"
+                :router-links="true" />
+            <media-element
+                :class="$style['c-loyaltyHeader-media']"
+                :text-size="fontSize"
+                :title="$t('header.title')"
+                :text="$t('header.text')"
+                :image-url="getImageBasedOnScreenSize"
+                image-align="center"
+                :flex="flexLayout"
+                :alt="$t('header.alt')"
+            />
+        </div>
+    </div>
+</template>
+
+<script>
+import BreadCrumbs from '@justeat/f-breadcrumbs';
+import MediaElement from '@justeat/f-media-element';
+import '@justeat/f-media-element/dist/f-media-element.css';
+import '@justeat/f-breadcrumbs/dist/f-breadcrumbs.css';
+import debounce from 'lodash.debounce';
+
+// this is used to define point at which image is swapped
+const REQUIRED_WINDOW_WIDTH = 600;
+const TEMP_IMG_PATH = 'https://d30v2pzvrfyzpo.cloudfront.net/b/hw/img/decoration/';
+
+export default {
+    name: 'StampcardsHeader',
+
+    components: {
+        BreadCrumbs,
+        MediaElement
+    },
+
+    data: () => ({
+        links:  [
+            {
+                name: 'Home',
+                url: '/'
+            },
+            {
+                name: 'For You',
+                url: '/offers'
+            },
+            {
+                name: 'StampCards',
+                url: '/offers/stamp-cards'
+            }
+        ],
+        fontSize: 'lg',
+        windowWidth: REQUIRED_WINDOW_WIDTH
+    }),
+
+    computed: {
+        /**
+         * Returns the image based on the current window size
+         * @returns {string}
+         */
+        getImageBasedOnScreenSize () {
+            return this.windowWidth >= REQUIRED_WINDOW_WIDTH
+                ? `${TEMP_IMG_PATH}stampcards-header-${this.$t('percentage')}.svg`
+                : `${TEMP_IMG_PATH}stampcards-header-small.svg`;
+        },
+
+        /**
+         * Sets the media element font size based on the current screen size
+         * @returns {string}
+         */
+        getFontSizeBasedOnScreenSize () {
```

Comment:
Does this get used atm? Can't see where it's called atm

Reply:
good spot, I missed adding it to media element.